source/openqasm/openqasm3/ast.py

```diff
@@ -1126,7 +1032,7 @@ class Pragma(Statement):
     Pragma
     Example::
 
-        #pragma {verbatim;}
+        #pragma val1 val2 val3
```

Comment:
Oops, this is a function of how I made the PR, thanks - I actually made the full swap over including your new pragmas and annotations (because originally I wasn't going to do a total rewrite of the grammar), and then "undid" those into a separate commit.  I must have left this documentation-only change in the wrong commit.

Reply:
Reverted in dee7a32.